pkgs/applications/misc/easystroke/default.nix

```
@@ -0,0 +1,31 @@
+# based on easystroke-git from Arch AUR
+# as found on https://github.com/teu5us/easystroke-nix
+
+{ pkgs, ... }:
+
+with pkgs;
+
+stdenv.mkDerivation rec {
+  name = "easystroke";
+  version = "patched-2018-12-05";
+  src = fetchTarball {
+    url = "https://github.com/thjaeger/easystroke/archive/refs/tags/0.6.0.tar.gz";
+    sha256 = "0kgzmwxqgbq3cgp2rvj09j9rk6h7ynbibim4mins3wzalgwy68dv";
+  };
```

```suggestion
  src = fetchFromGitHub {
    owner = "thjaeger";
    repo = "easystroke";
    rev = "0.6.0";
    sha256 = "0kgzmwxqgbq3cgp2rvj09j9rk6h7ynbibim4mins3wzalgwy68dv";
  };
```
Per guidelines, the most specific function should be used. Also note that `fetchTarball` is not meant to be used to fetch sources anyway.

---

Please subsitute the version.